docs/api-reference/create-next-app.md

```diff
@@ -56,14 +56,26 @@ Options:
   --eslint
 
     Initialize with ESLint config.
+    
+  --no-eslint
```

Comment:
Can we instead of listing each variant with `--no` have a note at the top or bottom saying that any option can be negated by prefixing with `--no` e.g. `--no-eslint`?

Reply:
Yea, I think it makes sense - I will do that.

Also, to improve devex  and make it easy to copy-paste I would still show full switch names for both.

let me get back to tomorrow 👍